app/controllers/concerns/bootstrappable.rb

```diff
@@ -8,13 +8,18 @@ module Bootstrappable
   private
 
   def bootstrap(data = {})
-    # Adding a `nonce` ensures that the bootstrap data is always unique, which is useful because
-    # that ensures that turbo will always evaluate the `<script>` tag(s) within which it's wrapped.
-    @bootstrap_data ||= {
-      nonce: SecureRandom.uuid,
-      current_user: current_user && UserSerializer::Basic.new(current_user),
-    }
-    @bootstrap_data.merge!(data)
+    if data.present?
```

Comment:
We used to always render bootstrap data, even when it wasn't needed/used. With this change, we can now check whether `@bootstrap_data` is present, and, if not, skip rendering it altogether.